less/navigation.less

```diff
@@ -1 +1,93 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.main-navigation {
+  width: 100%;
+  background: @nav-bg;
+  display: flex;
+  -webkit-animation-name: example; /* Safari 4.0 - 8.0 */
+  -webkit-animation-duration: 4s; /* Safari 4.0 - 8.0 */
+  animation-name: example;
+  animation-duration: 4s;
+
+  @keyframes example {
+    from {background-color:@nav-bg;}
+    to {background-color: yellow;
+    }
+}
+    
+  
+    .nav-container {
+      display: flex;
+      align-items: center;
+      flex-direction: column;
+      justify-content: space-between;
+      background: @nav-bg;
+      -webkit-animation-name: example; /* Safari 4.0 - 8.0 */
+      -webkit-animation-duration: 4s; /* Safari 4.0 - 8.0 */
+      animation-name: example;
+      animation-duration: 4s;
+
+      @keyframes example {
```

Comment:
Nice animation on the nav bar 🔥 You could make it into a sticky top by adding position: fixed to your nav container here 😊